### ISSUE TITLE
Flatpak browser all end up with the same executable_path

### ISSUE THREAD

**Issue by ptillemans** [Browsers-software/browsers#188]
**Describe the bug**
All installed flatpak browser (and browser apps like slack and spotify) end up with the same executable path "@@" which makes applying rules very hard

**System:**
 - OS: TUXEDO OS 3 x86_64
 - Browsers : all flatpak : Chrome, Brave, Firefox, (also Slack and Spotify for some reason).

The reason is that the logic to find something sufficiently unique takes the last word not starting with a '-' or '%' to avoid options and magic codes of desktop files. However the flatpak runtime has a feature to do file forwarding with the --file-forward option which replaces all files between @', confusing the best guess method.

I patched locally a version to fix this issue for my use case with this patch:

```
diff --git a/src/linux/linux_utils.rs b/src/linux/linux_utils.rs
index 612c6ea..7d7f19b 100644
--- a/src/linux/linux_utils.rs
+++ b/src/linux/linux_utils.rs
@@ -211,7 +211,11 @@ impl OsHelper {
         //  - to identify which Firefox profiles are allowed for firefox instance, they hash the binary path
         let executable_path_best_guess = command_parts
             .iter()
-            .rfind(|component| !component.starts_with("%") && !component.starts_with("-"))
+            .rfind(|component| {
+                !component.starts_with("%")
+                    && !component.starts_with("-")
+                    && !component.starts_with("@")
+            })
             .map(|path_perhaps| Path::new(path_perhaps))
             .unwrap_or(Path::new("unknown"));
```

This is a quick fix and probably needs special casing for the flatpak situation but for me the likelihood of confusion with real uses of @ is low enough. 

Great project, better that what I was looking for.

**Comment by liias** [comment]
Thank you very much for the report and figuring out the issue and fixing it!
I applied your fix to main branch as it's definitely better than before.

Fyi, not necessarily planning a release in July yet.

**Comment by liias** [comment]
Released in version 0.5.6
